BridgeAppSDK/Resources/SBAResourceFinder.swift

```diff
@@ -82,12 +82,41 @@ public class SBAResourceFinder: NSObject {
     }
     
     public func htmlNamed(resourceNamed: String) -> String? {
-        if let data = self.dataNamed(resourceNamed, ofType: "html") {
-            return String(data: data, encoding: NSUTF8StringEncoding)
+        if let data = self.dataNamed(resourceNamed, ofType: "html"),
+            let html = String(data: data, encoding: NSUTF8StringEncoding) {
+            return importHTML(html)
         }
         return nil
     }
     
+    func importHTML(input: String) -> String {
+        
+        // setup string
+        var html = input
+        func search(str: String, _ range: Range<String.Index>?) -> Range<String.Index>? {
+            return html.rangeOfString(str, options: .CaseInsensitiveSearch, range: range, locale: nil)
+        }
+        
+        // search for <import href="resourceName.html" /> and replace with contents of resource file
```

Comment:
does iOS still not support <link rel="import" href="...">? (see e.g. http://www.html5rocks.com/en/tutorials/webcomponents/imports/)